ciphers/monoalphabetic_cipher.cpp

```diff
@@ -0,0 +1,50 @@
+#include <bits/stdc++.h>
+using namespace std;
```

Comment:
These are discouraged. Please use each header/library you need and strictly use `std::`.